docs/usage/security/openidconnect-presets.md

```diff
@@ -4,6 +4,8 @@ title: OpenIDConnect Presets
 
 # ClusterOpenIDConnectPreset and OpenIDConnectPreset
 
+> **Note:** OpenID Connect is deprecated in favor of [Structured Authentication configuration](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#using-authentication-configuration). Setting OpenID Connect configurations is forbidden for clusters with Kubernetes version `>= 1.32`. 
```

Comment:
Nice, so we should also mark these resources as deprecated and drop them once we drop support for Kubernetes < 1.32. Can you add according statements?